openqabot/types/baseconf.py

```diff
@@ -21,6 +21,9 @@ def __call__(
     ) -> List[Dict[str, Any]]:
         pass
 
-    @abstractstaticmethod
+    @staticmethod
     def normalize_repos(config):
         pass
+
+    def filter_embargoed(self) -> bool:
+        return bool(self.settings.get(BaseConf.OBEY_EMBARGOED_CONF, False))
```

Comment:
same problem as before, don use `settings` for bot configuration --> `settings` is mean to be passed into openQA.

Reply:
ok will change that